### DISCUSSION TITLE
v1.9/1/2 not found

### DISCUSSION THREAD

**Discussion by gavinbaumanis** [sbt/sbt-native-packager#1433] [General]
Hi there when I try to use version 1.9.0 or newer I get a file not found error while trying to update.

`[error]   not found: C:\Users\gavin\.ivy2\localcom.typesafe.sbt\sbt-native-packager\scala_2.12\sbt_1.0\1.9.1\ivys\ivy.xml
[error]   not found: https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/com.typesafe.sbt/sbt-native-packager/scala_2.12/sbt_1.0/1.9.1/ivys/ivy.xml
[error]   not found: https://repo.typesafe.com/typesafe/ivy-releases/com.typesafe.sbt/sbt-native-packager/scala_2.12/sbt_1.0/1.9.1/ivys/ivy.xml`

1.8.1 and previous version seem to work fine.

**Comment by muuki88** [comment]
Due to the sun set of bintray I had to change the maven coordinates.

Readme and docs should be updated.

**Reply by gavinbaumanis** [reply]
Thanks very much... I didn't even think to check the readme!
All working now!
